README.md

```diff
@@ -2,9 +2,13 @@
 ### Required
 #### Ponder and answer these questions as you work through the other material for this week.
 - What is a framework?
+An abstraction that provides application-specific software that can be selectively changed by users.
 - What's the difference between a programming language and a framework?
+A programming language is the bones of a framework. It makes up applications, libraries, etc...  A framework is like a library in that it cuts a large deal of coding out of your work and sets the stage for doing lots of ...things?... quicker and easier.  That seems to make sense right?
 - What’s the difference between a library and a framework?
+A library is a group of functions and objects that serve a specific purpose while a framework is a collection of libraries that have more broad uses generally made for applications
```

Comment:
libraries = reusable collection of methods/functions which makes development easier and faster
frameworks = is the glue that holds everything together